target/linux/generic/files/drivers/net/phy/rtl8366_smi.h

```diff
@@ -64,6 +64,7 @@ struct rtl8366_smi {
 	u8			dbg_vlan_4k_page;
 #endif
 	struct mii_bus		*ext_mbus;
+	int			phy_id;
```

Comment:
`u32` or `unsigned int`. And I suggest moving it up.

Reply:
Done